app/src/main/java/org/schabi/newpipe/util/external_communication/TextLinkifier.java

```diff
@@ -33,7 +33,8 @@
 public final class TextLinkifier {
     public static final String TAG = TextLinkifier.class.getSimpleName();
 
-    private static final Pattern HASHTAGS_PATTERN = Pattern.compile("(#[A-Za-z0-9_]+)");
+    private static final Pattern HASHTAGS_PATTERN =
+            Pattern.compile("(#[A-Za-z0-9_\\u0080-\\uFFFF]+)");
```

Comment:
How does it handle a text with multiple hashtags in it like `#tag#noSpaceAnotherTag`

Reply:
It will treat it as two seperate hashtags #tag and #noSpaceAnotherTag. This is how it worked before and is how it is treated by Youtube.

Reply:
Outdated